package.json

```diff
@@ -46,6 +46,7 @@
     "@types/mustache": "^4.2.5",
     "async-retry": "^1.3.3",
     "hono": "^3.12.0",
+    "ts-node": "^10.9.2",
```

Comment:
Hi! What is the exact error that you are facing without adding ts-node package? Ideally, this package should not be a part of main dependencies.

Reply:
Hey @VisargD , I was not able to reproduce the issue somehow, so removed the package-lock json file and ts-node.

Reply:
I think you removed the entire `package-lock.json` file. Can you please revert both `package.json` and `package-lock.json` to the state they are in on the main branch?

Reply:
@VisargD I reverted all the changes for package-lock and package.sjon. Please check 😁